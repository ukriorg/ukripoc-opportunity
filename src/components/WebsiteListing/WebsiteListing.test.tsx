import React from "react";
import renderer from "react-test-renderer";
import { WebsiteListing } from "./index";

// describe("WebsiteListing", () => {
//     it("renders correctly", () => {
//         const tree = renderer
//             .create(<WebsiteListing addWebsiteListing={() => {}} />)
//             .toJSON();
//         expect(tree).toMatchSnapshot();
//     });
// });
