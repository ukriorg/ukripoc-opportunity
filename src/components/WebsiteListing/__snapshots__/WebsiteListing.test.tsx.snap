// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebsiteListing renders correctly 1`] = `
<h1
  className="sc-kGXeez fysBMX sc-kgoBCf dSnglf"
  size="XLARGE"
>
  Not found
</h1>
`;
